api/api.go

```diff
@@ -70,6 +70,12 @@ type RHPScanRequest struct {
 	HostIP  string    `json:"hostIP"`
 }
 
+// RHPScanResponse is the response type for the /rhp/scan endpoint.
+type RHPScanResponse struct {
+	rhpv2.HostSettings
+	Ping int64 `json:"ping"`
```

Comment:
I can see why reporting milliseconds would be nicer for interop than a raw `time.Duration`, but maybe we should make it a `float64`, or a custom type? As we add more metrics, these duration values will become very common, so perhaps a custom type is warranted.

Also, I prefer to avoid embedding except where absolutely necessary. In this case, I think we want it to be clear that `ping` is not a host setting :)

Reply:
Are you suggesting to add something like

```go
type RequestDuration struct {
	Amount float64  `json:"amount"`
	Unit   string `json:"unit"`
}
```
?

I thought about that but I feel `ping` is implied to be `ms`... in other metrics cases I do agree it would be useful to have a custom type but in the case of `ping` I feel it would be weird?

Reply:
I'm thinking:
```go
// A Duration is the elapsed time between two instants. Durations are encoded as
// an integer number of milliseconds.
type Duration time.Duration

func (d Duration) MarshalText() ([]byte, error) { ... }
func (d *Duration) UnmarshalText(b []byte) error { ... }
```
so `Duration(123 * time.Millisecond)` would be marshaled as `123` (or `"123"` in JSON).

Reply:
updated